.github/workflows/continuous-delivery.yml

```diff
@@ -82,61 +103,26 @@ jobs:
 
       ## Deploy
       - name: Deploy packages
-        run: mvn -B deploy -DskipTests -Dmaven.install.skip=true
+        run: mvn ${{ env.MAVEN_CLI_OPTS }} deploy -Dmaven.install.skip=true
 
+      ## Create Release
       - name: Create Release
         id: create_release
-        uses: actions/create-release@v1
-        env:
-          #don't use the GITHUB_TOKEN https://github:community/t/actions-not-triggered-on-release/17944/12:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: softprops/action-gh-release@v1
         with:
+          name: Release ${{ env.NEW_VERSION }}
           tag_name: ${{ steps.semanticversion.outputs.new_tag }}
-          release_name: ${{ steps.semanticversion.outputs.new_version }}
-
+          draft: false
+          prerelease: false
 
```

Comment:
da fehlen ja noch die Dateien

Reply:
versions.xml gibt es hier ja nicht, daher hab ich es so gemacht wie in core

Reply:
In Zeile 98 bis 133 werden ja noch 3 Dateien (jar, javadoc-jar und sources-jar) mit ans Release gehängt, das kann man mit dem softprops/action-gh-release@v1 so wie ich das gelesen habe ja auch machen.

Reply:
Ich hab Source und javadoc als Asset angegeben. Dabei hab ich die Pfade aus den Zeilen 98-133 benutzt.Ich hoff das passt so

Reply:
Laut docu sieht das sehr gut aus, allerdings fehlt noch die .jar selbst ;)